contracts/contracts/strategies/plume/RoosterAMOStrategy.sol

```diff
@@ -1083,8 +1084,9 @@ contract RoosterAMOStrategy is InitializableAbstractStrategy {
     }
 
     /**
-     * @notice Tick dominance denominated in 1e18
-     *
+     * @notice Tick dominance   denominated in 1e18
+     * @return _tickDominance   The share of liquidity in TICK_NUMBER tick owned
+     * by the strategy contract denominated in 1e18
```

Comment:
nit: When using multiline comments, we usually leave some spaces at the start of other line in other places like this:
```
    /**
     * @notice Tick dominance   denominated in 1e18
     * @return _tickDominance   The share of liquidity in TICK_NUMBER tick owned
     *                          by the strategy contract denominated in 1e18
     */
     ```